src/NodaTime.Demo/OffsetDemo.cs

```diff
@@ -62,6 +62,58 @@ public void ConstructionFromTimeSpan()
             Assert.AreEqual(5400, offset.Seconds);
         }
 
+        [Test]
+        public void Add()
+        {
+            var lefHandOffset = Offset.FromHours(5);
+            var rightHandOffset = Offset.FromHours(6);
+
+            var result = Snippet.For(Offset.Add(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset + rightHandOffset);
+        }
+
+        [Test]
+        public void Substract()
+        {
+            var lefHandOffset = Offset.FromHours(7);
+            var rightHandOffset = Offset.FromHours(5);
+
+            var result = Snippet.For(Offset.Subtract(lefHandOffset, rightHandOffset));
+
+            Assert.AreEqual(result, lefHandOffset - rightHandOffset);
+        }
+
+        [Test]
+        public void CompareTo()
+        {
+            var smallerOffset = Offset.FromHours(3);
+            var largerOffset = Offset.FromHours(5);
+            var equalOffset = Offset.FromHours(5);
+
+            var lessThan = smallerOffset.CompareTo(largerOffset);
+            var equalTo = largerOffset.CompareTo(equalOffset);
+            var largerThan = largerOffset.CompareTo(smallerOffset);
+
+            Assert.Less(lessThan, 0);
```

Comment:
This won't currently work (IIRC) - although I could make it work.

Reply:
I think it would be beneficial to make these work. :)

Reply:
Okay, I'll try to do that tomorrow.

Reply:
I may have miscommunicated before - I think it's still worth doing greater than and equal to - you just don't need to include Snippet.For multiple times. So:

```csharp
 var smallerOffset = Offset.FromHours(3);
var largerOffset = Offset.FromHours(5);

var lessThan = Snippet.For(smallerOffset.CompareTo(largerOffset));
var equal = smallerOffset.CompareTo(smallerOffset);
var greaterThan = largerOffset.CompareTo(smallerOffset);

Assert.Less(lessThan, 0);
Assert.AreEqual(0, equal);
Assert.Less(greaterThan, 0);
```